src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
-body {
+@import "./variables";
+
+div,
+a,
+p,
+h2,
+img {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
```

Comment:
2 [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

Reply:
Please fix the comment (hint: check font-family)